gtfu/src/main/java/gtfu/Util.java

```diff
@@ -969,4 +971,15 @@ public boolean verify(String arg0, SSLSession arg1) {
         HttpsURLConnection.setDefaultHostnameVerifier(validHosts);
     }
 
+    public static boolean isValidJSON(String jsonString){
+        try{
+            ObjectMapper objectMapper = new ObjectMapper();
+            JsonNode jsonNode = objectMapper.readTree(jsonString);
+        } catch(JsonProcessingException e){
+            Debug.log("* Invalid JSON detected. Stack trace:");
+            e.printStackTrace();
```

Comment:
<pedantery>

`e.printStackTrace()` writes to stdout. `Debug.log()` currently also logs to stdout, but that may change in the future (different or additional streams).  To go the extra mile, we could use `printStackTrace(PrintStream)` with a `ByteArrayOutputStream` backed `PrintStream` and then convert that BAOS to a string and pass to `Debug.log()`.

This is not required at this point! You can just paste something along the blurb above into the code for now.

</pedantery>

Reply:
To get extra fancy, we could upgrade `Debug.log()` to check the type of the arg. If it's of type `Throwable` then perform the steps outlined above. That way, things would just automagically work.

Reply:
To continue the rambling, we can add `Debug.setStackTraceForExceptionArg(boolean)`, which would control whether or not traces are generated. The default should probably be `true`.

Reply:
Thank you! Added comment about this